src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationSealedClasses.java

```diff
@@ -0,0 +1,40 @@
+//non-compiled with javac: Compilable with Java21                           //indent:0 exp:0
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;     //indent:0 exp:0
+
+
+/* Config:                                                                  //indent:0 exp:0
+ * basicOffset = 4                                                          //indent:1 exp:1
+ * braceAdjustment = 0                                                      //indent:1 exp:1
+ * caseIndent = 4                                                           //indent:1 exp:1
+ * tabWidth = 4                                                             //indent:1 exp:1
+ * throwsIndent = 8                                                         //indent:1 exp:1
+ * forceStrictCondition = true                                              //indent:1 exp:1
+ */                                                                         //indent:1 exp:1
+public class InputIndentationSealedClasses { //indent:0 exp:0
+sealed //indent:0 exp:4 warn
+ class Test //indent:1 exp:4 warn
+     permits A, B { } //indent:5 exp:4 warn
```

Comment:
All the invalid examples should have duplicates showing their valid ones. It is only with both do we know that violations we create can be resolved, which this check is known to have issues with.

Reply:
done